dali/python/nvidia/dali/pipeline.py

```diff
@@ -1455,7 +1455,82 @@ def _discriminate_args(func, **func_kwargs):
     return ctor_args, fn_args
 
 
-def pipeline_def(fn=None, **pipeline_kwargs):
+def _regroup_args(func, pipeline_def_kwargs, fn_call_kwargs):
+    """Regroup arguments that are directed into Pipeline object construction and those that
+    are passed into pipeline definition function.
+
+    Parameters
```

Comment:
I know this doc implies it, but please map explicitly return value with its definition. Like:
```
Regroup arguments that are directed into Pipeline object construction (Pipeline kwargs) and those that are passed into pipeline definition function (Function kwargs).
```

Reply:
Done